jobs/mobile-developer.md

```diff
@@ -0,0 +1,57 @@
+
+
+## How the job looks
+
+- We have jobs both for Android and iOS developers, experienced (or interested to learn) in React Native. We primarily use Kotlin for Android and Swift for iOS, yet certain Java or Objective-C knowledge might be useful.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually our projects last for about 1.5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- Salary 𝚫: 24k—48k USD.
+- For development we usually use Kanban or similar process in Trello, Pivotal of Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use [Holacracy](https://en.wikipedia.org/wiki/Holacracy) for organizational governance.
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## How a good candidate looks
+
+- You have solid experience either in Android or iOS.
+- You wish to work with React Native, even if you have not much experience in it.
+- You have an opinion on how to architect the app and can compare MVP with MVVM.
+- You know how different components of mobile OSes work: push notifications, keystore & keychain, background tasks, GPS api, etc.
+- Have experience releasing apps to stores and understand the whole process from building to signing to publishing.
+- You value high-quality maintainable code: know how organize codebase the way when something changes in API we don't need to rewrite a lot of code across the app.
+- Have experience writing end-to-end and unit tests and maintain high coverage.
+- Your English skills allow you to feel comfortable communicating in it (verbally and written) the whole working day.
+- You know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication.
+- Learning process is an important part of your professional activities.
+- Teamwork is not an abstract word for you, but a set of principles and agreements you follow.
+
+
+## What you'll definitely need to work with us
+
+- Experience in writing native mobile apps for Android or iOS.
+- Experience in working with React Native or desire to learn it.
+- English knowledge to communicate with colleagues and clients.
+- Communication skills to collaborate remotely.
+- Independence and initiative - we won't control how you work, but we expect the work to be done.
```

Comment:
@maryili, we maybe create a false representation that people in the team work independently but in fact I encourage collaboration with teammates, pair- and mob-programming, and technical discussions.